.github/workflows/bump-otel-version.yml

```diff
@@ -0,0 +1,47 @@
+name: bump-otel-version
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '24 5 * * 1-5'
+
+permissions:
+  contents: read
+
+env:
+  JOB_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+jobs:
+  update-otel:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: ["main", "8.19"]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ matrix.branch }}
```

Comment:
This will enforce to have `.ci/updatecli/update-otel.yml` up to date, is that the expected behaviour? Or do you prefer maintaining the updatecli pipeline in `main` only? If so, then `ref...` is not needed